debian/rules

```diff
@@ -23,6 +24,21 @@ else
 endif
 endif
 
+override_dh_gencontrol:
+ifeq ($(DIST_RELEASE),trusty)
+	echo "extra:Depends=libapt-pkg4.12 (>= 1.0.1ubuntu2.23)" >> debian/ubuntu-advantage-tools.substvars
+else ($(DIST_RELEASE),xenial))
+	echo "extra:Depends=libapt-pkg5.0 (>= 1.2.30)" >> debian/ubuntu-advantage-tools.substvars
+else ($(DIST_RELEASE),cosmic))
+	echo "extra:Depends=libapt-pkg5.0 (>= 1.7.3)" >> debian/ubuntu-advantage-tools.substvars
+else ($(DIST_RELEASE),disco))
+	echo "extra:Depends=libapt-pkg5.0 (>= 1.8.0~rc1)" >> debian/ubuntu-advantage-tools.substvars
+else ($(DIST_RELEASE),eoan))
+	echo "extra:Depends=libapt-pkg5.0 (>= 1.8.1)" >> debian/ubuntu-advantage-tools.substvars
+endif
+	dh_gencontrol $@
```

Comment:
I'm checking if `$@` is necessary, it looks like it isn't (I know I suggested it).

Reply:
Just confirmed, `$@`  is unecessary. `dh_gencontrol` by itself is sufficient.

Reply:
dropped in 833263fe1909e4ad865a85da9a5b5cd470c8f641